ci: investigate gofmt test flake

During the test of commit https://github.com/myitcv/govim/commit/7034f5a67ee5043c50231e15d78d4cf23c8ca614 in Travis one of the Jobs failed to run `gofmt.txt`.
Rerunning that job made it pass.

Job:
<code>GO_VERSION="go1.12.7" VIM_FLAVOR="gvim" VIM_VERSION="v8.1.1657" VIM_COMMAND="xvfb-run -a gvim -f"
</code>

Failed test:
<pre>
> go test $(go list ./... | grep -v 'govim/internal/golang_org_x_tools')
ok      github.com/myitcv/govim 10.199s
--- FAIL: TestScripts (22.05s)
    --- FAIL: TestScripts/scripts (0.00s)
        --- FAIL: TestScripts/scripts/gofmt (1.27s)
            testscript.go:308:
                # Test that govim#config#Set("FormatOnSave", "gofmt") and GOVIMGoFmt work
                # :GOVIMGoFmt whole file (0.304s)
                # Format on save (0.064s)
                # Format on save (bad syntax) (0.152s)
                > cp file.go.bad file.go
                > vim ex 'e! file.go'
                > errlogmatch -wait 30s 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/file.go
                > vim ex 'w'
                > cmp file.go file.go.bad
                > vim expr 'getqflist()'
                [stdout]
                [{"bufnr":1,"col":6,"lnum":5,"module":"","nr":0,"pattern":"","text":"undeclared name: os","type":"","valid":1,"vcol":0}]
                > stdout '^\Q[{"bufnr":1,"col":1,"lnum":3,"module":"","nr":0,"pattern":"","text":"expected declaration, found blah","type":"","valid":1,"vcol":0}]\E$'
                FAIL: testdata/gofmt.txt:28: no match for `^\Q[{"bufnr":1,"col":1,"lnum":3,"module":"","nr":0,"pattern":"","text":"expected declaration, found blah","type":"","valid":1,"vcol":0}]\E$` found in stdout

FAIL
</pre>

Full travis log - failed run:
<details><pre>
travis_fold:start:worker_info
Worker information
hostname: 32dfedda-76bb-4c32-85a3-74586a7c2452@1.worker-org-78c6f46d9c-rjbcp.gce-production-1
version: v6.2.1 https://github.com/travis-ci/worker/tree/4e3246c044eb4915c2378ffacd0b3d3ed0136bba
instance: travis-job-57acc76f-6e4a-4ecb-934a-9dc9682f5805 travis-ci-sardonyx-xenial-1553530528-f909ac5 (via amqp)
startup: 6.349728682s
travis_fold:end:worker_info
travis_fold:start:system_info
Build system information
Build language: generic
Build group: stable
Build dist: xenial
Build id: 564325663
Job id: 564325674
Runtime kernel version: 4.15.0-1028-gcp
travis-build version: 24c88d12b
Build image provisioning date and time
Mon Mar 25 16:43:24 UTC 2019
Operating System Details
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.6 LTS
Release:	16.04
Codename:	xenial
Systemd Version
systemd 229
Cookbooks Version
42e42e4 https://github.com/travis-ci/travis-cookbooks/tree/42e42e4
git version
git version 2.21.0
bash version
GNU bash, version 4.3.48(1)-release (x86_64-pc-linux-gnu)
gcc version
gcc (Ubuntu 5.4.0-6ubuntu1~16.04.11) 5.4.0 20160609
docker version
Client:
 Version:           18.06.0-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        0ffa825
 Built:             Wed Jul 18 19:11:02 2018
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.0-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       0ffa825
  Built:            Wed Jul 18 19:09:05 2018
  OS/Arch:          linux/amd64
  Experimental:     false
clang version
clang version 7.0.0 (tags/RELEASE_700/final)
jq version
jq-1.5
bats version
Bats 0.4.0
shellcheck version
0.6.0
shfmt version
v2.6.3
ccache version
3.2.4
cmake version
cmake version 3.12.4
heroku version
heroku/7.22.7 linux-x64 node-v11.10.1
imagemagick version
Version: ImageMagick 6.8.9-9 Q16 x86_64 2018-09-28 http://www.imagemagick.org
md5deep version
4.4
mercurial version
version 4.8
mysql version
mysql  Ver 14.14 Distrib 5.7.25, for Linux (x86_64) using  EditLine wrapper
openssl version
OpenSSL 1.0.2g  1 Mar 2016
packer version
1.3.3
postgresql client version
psql (PostgreSQL) 10.7 (Ubuntu 10.7-1.pgdg16.04+1)
ragel version
Ragel State Machine Compiler version 6.8 Feb 2013
sudo version
1.8.16
gzip version
gzip 1.6
zip version
Zip 3.0
vim version
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Nov 24 2016 16:44:48)
iptables version
iptables v1.6.0
curl version
curl 7.47.0 (x86_64-pc-linux-gnu) libcurl/7.47.0 GnuTLS/3.4.10 zlib/1.2.8 libidn/1.32 librtmp/2.3
wget version
GNU Wget 1.17.1 built on linux-gnu.
rsync version
rsync  version 3.1.1  protocol version 31
gimme version
v1.5.3
nvm version
0.34.0
perlbrew version
/home/travis/perl5/perlbrew/bin/perlbrew  - App::perlbrew/0.86
phpenv version
rbenv 1.1.2
rvm version
rvm 1.29.7 (latest) by Michal Papis, Piotr Kuczynski, Wayne E. Seguin [https://rvm.io]
default ruby version
ruby 2.5.3p105 (2018-10-18 revision 65156) [x86_64-linux]
CouchDB version
couchdb 1.6.1
ElasticSearch version
5.5.0
Installed Firefox version
firefox 63.0.1
MongoDB version
MongoDB 4.0.7
PhantomJS version
2.1.1
Pre-installed PostgreSQL versions
9.4.21
9.5.16
9.6.12
Redis version
redis-server 5.0.4
Pre-installed Go versions
1.11.1
mvn version
Apache Maven 3.6.0 (97c98ec64a1fdfee7767ce5ffb20918da4f719f3; 2018-10-24T18:41:47Z)
gradle version
Gradle 4.10.2!
lein version
Leiningen 2.9.1 on Java 11.0.2 OpenJDK 64-Bit Server VM
Pre-installed Node.js versions
v10.15.3
v11.0.0
v4.9.1
v6.17.0
v8.12.0
v8.15.1
v8.9
phpenv versions
  system
  5.6
  5.6.40
  7.1
  7.1.27
  7.2
* 7.2.15 (set by /home/travis/.phpenv/version)
  hhvm
  hhvm-stable
composer --version
Composer version 1.8.4 2019-02-11 10:52:10
Pre-installed Ruby versions
ruby-2.3.8
ruby-2.4.5
ruby-2.5.3
travis_fold:end:system_info

travis_fold:start:docker_mtu
travis_fold:end:docker_mtu
travis_fold:start:resolvconf
travis_fold:end:resolvconf

travis_fold:start:git.checkout
travis_time:start:0f6ca9fc
$ git clone --depth=50 --branch=master https://github.com/myitcv/govim.git myitcv/govim
Cloning into 'myitcv/govim'...
travis_time:end:0f6ca9fc:start=1564217309117517681,finish=1564217310082598523,duration=965080842
$ cd myitcv/govim
$ git checkout -qf 7034f5a67ee5043c50231e15d78d4cf23c8ca614
travis_fold:end:git.checkout


Setting environment variables from repository settings
$ export GH_TOKEN=[secure]
$ export GH_USER=myitcv

Setting environment variables from .travis.yml
$ export GO111MODULE=on
$ export GOPROXY=https://proxy.golang.org
$ export GO_VERSION="go1.12.7"
$ export VIM_FLAVOR="gvim"
$ export VIM_VERSION="v8.1.1657"
$ export VIM_COMMAND="xvfb-run -a gvim -f"

$ bash -c 'echo $BASH_VERSION'
4.3.48(1)-release

travis_fold:start:before_install.1
travis_time:start:1261f8c8
$ docker --version
Docker version 18.06.0-ce, build 0ffa825
travis_time:end:1261f8c8:start=1564217310124587151,finish=1564217310446463265,duration=321876114
travis_fold:end:before_install.1
travis_fold:start:before_install.2
travis_time:start:1626d87e
$ ./_scripts/buildGovimImage.sh
Sending build context to Docker daemon  2.048kB


Step 1/11 : FROM govim/govim:go1.12.7_gvim_v8.1.1657_v1
go1.12.7_gvim_v8.1.1657_v1: Pulling from govim/govim
898c46f3b1a1: Pulling fs layer
63366dfa0a50: Pulling fs layer
041d4cd74a92: Pulling fs layer
6e1bee0f8701: Pulling fs layer
930c33e06246: Pulling fs layer
c2eb65575a45: Pulling fs layer
0adb2cbf93a0: Pulling fs layer
0650c35c0db6: Pulling fs layer
7c859565a35f: Pulling fs layer
ce8f93ab5d45: Pulling fs layer
27e2dca98e6c: Pulling fs layer
36b19e4bf3f2: Pulling fs layer
afd947b29ffc: Pulling fs layer
df7a3dc01368: Pulling fs layer
2c21a8530cff: Pulling fs layer
ea566e43f97e: Pulling fs layer
7c859565a35f: Waiting
ce8f93ab5d45: Waiting
27e2dca98e6c: Waiting
36b19e4bf3f2: Waiting
afd947b29ffc: Waiting
df7a3dc01368: Waiting
2c21a8530cff: Waiting
ea566e43f97e: Waiting
6e1bee0f8701: Waiting
930c33e06246: Waiting
c2eb65575a45: Waiting
0adb2cbf93a0: Waiting
0650c35c0db6: Waiting
63366dfa0a50: Verifying Checksum
63366dfa0a50: Download complete
041d4cd74a92: Verifying Checksum
041d4cd74a92: Download complete
6e1bee0f8701: Verifying Checksum
6e1bee0f8701: Download complete
898c46f3b1a1: Verifying Checksum
898c46f3b1a1: Download complete
930c33e06246: Verifying Checksum
930c33e06246: Download complete
c2eb65575a45: Verifying Checksum
c2eb65575a45: Download complete
0adb2cbf93a0: Verifying Checksum
0adb2cbf93a0: Download complete
ce8f93ab5d45: Verifying Checksum
ce8f93ab5d45: Download complete
0650c35c0db6: Verifying Checksum
0650c35c0db6: Download complete
7c859565a35f: Verifying Checksum
7c859565a35f: Download complete
36b19e4bf3f2: Verifying Checksum
36b19e4bf3f2: Download complete
afd947b29ffc: Verifying Checksum
afd947b29ffc: Download complete
df7a3dc01368: Verifying Checksum
df7a3dc01368: Download complete
ea566e43f97e: Verifying Checksum
ea566e43f97e: Download complete
898c46f3b1a1: Pull complete
63366dfa0a50: Pull complete
041d4cd74a92: Pull complete
6e1bee0f8701: Pull complete
930c33e06246: Pull complete
27e2dca98e6c: Verifying Checksum
27e2dca98e6c: Download complete
2c21a8530cff: Verifying Checksum
2c21a8530cff: Download complete
c2eb65575a45: Pull complete
0adb2cbf93a0: Pull complete
0650c35c0db6: Pull complete
7c859565a35f: Pull complete
ce8f93ab5d45: Pull complete
27e2dca98e6c: Pull complete
36b19e4bf3f2: Pull complete
afd947b29ffc: Pull complete
df7a3dc01368: Pull complete
2c21a8530cff: Pull complete
ea566e43f97e: Pull complete
Digest: sha256:d8a9e417d9b273970a5d76e687ddceec2961236fbc49318bc0e945b8e9997a16
Status: Downloaded newer image for govim/govim:go1.12.7_gvim_v8.1.1657_v1
 ---> 35bfc91d63e5
Step 2/11 : ARG USER
 ---> Running in a905ebd0fac6
Removing intermediate container a905ebd0fac6
 ---> ea2850ff9b41
Step 3/11 : ARG UID
 ---> Running in 763a4726c4d8
Removing intermediate container 763a4726c4d8
 ---> f6b36edb9131
Step 4/11 : ARG GID
 ---> Running in 9829d7d9ca46
Removing intermediate container 9829d7d9ca46
 ---> 1de5a7e5785b
Step 5/11 : ENV PATH=/vbash/bin:/home/$USER/.local/bin:$PATH
 ---> Running in 828eb8b6dc91
Removing intermediate container 828eb8b6dc91
 ---> b154389b7832
Step 6/11 : ENV GOPATH=/home/$USER/gopath
 ---> Running in 2a98be0cfb5e
Removing intermediate container 2a98be0cfb5e
 ---> 778cc80cdb13
Step 7/11 : RUN sh -c "if ! getent group $GID; then groupadd -g $GID $USER; fi" &&     adduser --uid $UID --gid $GID --disabled-password --gecos "" $USER
 ---> Running in ec0758d74d79
Adding user `travis' ...
Adding new user `travis' (2000) with group `travis' ...
Creating home directory `/home/travis' ...
Copying files from `/etc/skel' ...
Removing intermediate container ec0758d74d79
 ---> 02f9eff64f5d
Step 8/11 : RUN usermod -aG sudo $USER
 ---> Running in ff0911e70aae
Removing intermediate container ff0911e70aae
 ---> d826495f9f4b
Step 9/11 : RUN echo "$USER ALL=(ALL:ALL) NOPASSWD: ALL" > /etc/sudoers.d/$USER
 ---> Running in 289cfb9938de
Removing intermediate container 289cfb9938de
 ---> f0c9b32debcc
Step 10/11 : WORKDIR /home/$USER
 ---> Running in 3dbfb7a61a9f
Removing intermediate container 3dbfb7a61a9f
 ---> 766de807e843
Step 11/11 : USER $USER
 ---> Running in f81a055f70dd
Removing intermediate container f81a055f70dd
 ---> eaadf9515bfd
Successfully built eaadf9515bfd
Successfully tagged govim:latest
travis_time:end:1626d87e:start=1564217310452303258,finish=1564217393994212542,duration=83541909284
travis_fold:end:before_install.2
travis_time:start:29ff64a4
$ ./_scripts/runDockerRun.sh
> source "${BASH_SOURCE%/*}/common.bash"
> eval "VIM_COMMAND=\"\$DEFAULT_${VIM_FLAVOR^^}_COMMAND\""
> cat <<EOD
Environment is:
  VIM_FLAVOR=$VIM_FLAVOR
  VIM_COMMAND=$VIM_COMMAND
EOD
Environment is:
  VIM_FLAVOR=gvim
  VIM_COMMAND=xvfb-run -a gvim -f
> echo -e "machine api.github.com\n  login $GH_USER\n  password $GH_TOKEN" >>~/.netrc
> echo -e "machine github.com\n  login $GH_USER\n  password $GH_TOKEN" >>~/.netrc
> echo -e "machine githubusercontent.com\n  login $GH_USER\n  password $GH_TOKEN" >>~/.netrc
> go version
go version go1.12.7 linux/amd64
> vim --version
VIM - Vi IMproved 8.1 (2018 May 18, compiled Jul 18 2019 08:20:25)
Included patches: 1-1657
Compiled by root@3ccfdb568c5f
Huge version with GTK2 GUI.  Features included (+) or not (-):
+acl               -farsi             -mouse_sysmouse    -tag_any_white
+arabic            +file_in_path      +mouse_urxvt       -tcl
+autocmd           +find_in_path      +mouse_xterm       +termguicolors
+autochdir         +float             +multi_byte        +terminal
-autoservername    +folding           +multi_lang        +terminfo
+balloon_eval      -footer            -mzscheme          +termresponse
+balloon_eval_term +fork()            -netbeans_intg     +textobjects
+browse            +gettext           +num64             +textprop
++builtin_terms    -hangul_input      +packages          +timers
+byte_offset       +iconv             +path_extra        +title
+channel           +insert_expand     -perl              +toolbar
+cindent           +job               +persistent_undo   +user_commands
+clientserver      +jumplist          +postscript        +vartabs
+clipboard         +keymap            +printer           +vertsplit
+cmdline_compl     +lambda            +profile           +virtualedit
+cmdline_hist      +langmap           -python            +visual
+cmdline_info      +libcall           -python3           +visualextra
+comments          +linebreak         +quickfix          +viminfo
+conceal           +lispindent        +reltime           +vreplace
+cryptv            +listcmds          +rightleft         +wildignore
+cscope            +localmap          -ruby              +wildmenu
+cursorbind        -lua               +scrollbind        +windows
+cursorshape       +menu              +signs             +writebackup
+dialog_con_gui    +mksession         +smartindent       +X11
+diff              +modify_fname      -sound             -xfontset
+digraphs          +mouse             +spell             +xim
+dnd               +mouseshape        +startuptime       -xpm
-ebcdic            +mouse_dec         +statusline        +xsmp_interact
+emacs_tags        -mouse_gpm         -sun_workshop      +xterm_clipboard
+eval              -mouse_jsbterm     +syntax            -xterm_save
+ex_extra          +mouse_netterm     +tag_binary        
+extra_search      +mouse_sgr         -tag_old_static    
   system vimrc file: "$VIM/vimrc"
     user vimrc file: "$HOME/.vimrc"
 2nd user vimrc file: "~/.vim/vimrc"
      user exrc file: "$HOME/.exrc"
  system gvimrc file: "$VIM/gvimrc"
    user gvimrc file: "$HOME/.gvimrc"
2nd user gvimrc file: "~/.vim/gvimrc"
       defaults file: "$VIMRUNTIME/defaults.vim"
    system menu file: "$VIMRUNTIME/menu.vim"
  fall-back for $VIM: "/vim/share/vim"
Compilation: gcc -c -I. -Iproto -DHAVE_CONFIG_H -DFEAT_GUI_GTK  -pthread -I/usr/include/gtk-2.0 -I/usr/lib/x86_64-linux-gnu/gtk-2.0/include -I/usr/include/gio-unix-2.0/ -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libpng16 -I/usr/include/pango-1.0 -I/usr/include/harfbuzz -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/freetype2 -I/usr/include/libpng16   -g -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1       
Linking: gcc   -L/usr/local/lib -Wl,--as-needed -o vim   -lgtk-x11-2.0 -lgdk-x11-2.0 -lpangocairo-1.0 -latk-1.0 -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lpangoft2-1.0 -lpango-1.0 -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lSM -lICE -lXt -lX11 -lXdmcp -lSM -lICE  -lm -ltinfo -lnsl  -ldl           
> ./_scripts/revendorToolsInternal.sh
	> source "${BASH_SOURCE%/*}/common.bash"
	> shopt -s globstar
	> shopt -s extglob
	> go mod download
	go: finding golang.org/x/tools v0.0.0-20190716221150-e98af2309876
	go: finding github.com/kr/pty v1.1.5
	go: finding honnef.co/go/tools v0.0.0-20190614002413-cb51c254f01b
	go: finding golang.org/x/sys v0.0.0-20190429190828-d89cdac9e872
	go: finding github.com/myitcv/vbash v0.0.4
	go: finding golang.org/x/tools/gopls v0.1.4-0.20190716221150-e98af2309876
	go: finding github.com/acarl005/stripansi v0.0.0-20180116102854-5a71ef0e047d
	go: finding golang.org/x/sync v0.0.0-20190423024810-112230192c58
	go: finding gopkg.in/retry.v1 v1.0.3
	go: finding gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
	go: finding github.com/fsnotify/fsnotify v1.4.7
	go: finding golang.org/x/net v0.0.0-20190620200207-3b0461eec859
	go: finding github.com/rogpeppe/go-internal v1.3.0
	go: finding github.com/kr/pretty v0.1.0
	go: finding github.com/rogpeppe/go-internal v1.2.1
	go: finding github.com/frankban/quicktest v1.2.2
	go: finding mvdan.cc/sh/v3 v3.0.0-alpha1
	go: finding golang.org/x/tools v0.0.0-20190710153321-831012c29e42
	go: finding github.com/rogpeppe/clock v0.0.0-20190514195947-2896927a307a
	go: finding github.com/google/renameio v0.1.0
	go: finding github.com/kisielk/gotool v1.0.0
	go: finding github.com/BurntSushi/toml v0.3.1
	go: finding golang.org/x/mod v0.0.0-20190513183733-4bf6d317e70e
	go: finding golang.org/x/text v0.3.0
	go: finding golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2
	go: finding golang.org/x/tools v0.0.0-20190530171427-2b03ca6e44eb
	go: finding gopkg.in/errgo.v2 v2.1.0
	go: finding github.com/kr/text v0.1.0
	go: finding golang.org/x/sys v0.0.0-20190215142949-d0b11bdaac8a
	go: finding golang.org/x/sys v0.0.0-20181107165924-66b7b1311ac8
	go: finding golang.org/x/sync v0.0.0-20181108010431-42b317875d0f
	go: finding github.com/google/go-cmp v0.2.1-0.20190312032427-6f77996f0c42
	go: finding github.com/kr/pty v1.1.1
	go: finding golang.org/x/net v0.0.0-20190311183353-d8887717615a
	go: finding gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127
	go: finding golang.org/x/crypto v0.0.0-20190510104115-cbcb75029529
	go: finding golang.org/x/crypto v0.0.0-20181106171534-e4dc69e5b2fd
	go: finding golang.org/x/sys v0.0.0-20190412213103-97732733099d
	go: finding golang.org/x/net v0.0.0-20190404232315-eb5bcb51f2a3
	> tools=$(go list -m -f={{.Dir}} golang.org/x/tools)
	> echo "Tools is $tools"
	Tools is /home/travis/gopath/pkg/mod/golang.org/x/tools@v0.0.0-20190716221150-e98af2309876
	> cd $(git rev-parse --show-toplevel)
	> rsync -a --delete --chmod=D0755,F0644 $tools/internal/ ./cmd/govim/internal/golang_org_x_tools
	> sed -i 's+golang.org/x/tools/internal+github.com/myitcv/govim/cmd/govim/internal/golang_org_x_tools+g' ./cmd/govim/internal/golang_org_x_tools/**/*.go
	> rm ./cmd/govim/internal/golang_org_x_tools/**/*_test.go
	> rm -f ./cmd/govim/internal/golang_org_x_tools/LICENSE
	> cp $tools/LICENSE ./cmd/govim/internal/golang_org_x_tools
	> go mod tidy
> go install golang.org/x/tools/gopls
> rm -f $(git ls-files -- ':!:cmd/govim/internal/golang_org_x_tools' '**/gen_*.*' 'gen_*.*') .travis.yml
> go generate $(go list ./... | grep -v 'govim/internal/golang_org_x_tools')
> go test $(go list ./... | grep -v 'govim/internal/golang_org_x_tools')
ok  	github.com/myitcv/govim	10.199s
--- FAIL: TestScripts (22.05s)
    --- FAIL: TestScripts/scripts (0.00s)
        --- FAIL: TestScripts/scripts/gofmt (1.27s)
            testscript.go:308: 
                # Test that govim#config#Set("FormatOnSave", "gofmt") and GOVIMGoFmt work
                # :GOVIMGoFmt whole file (0.304s)
                # Format on save (0.064s)
                # Format on save (bad syntax) (0.152s)
                > cp file.go.bad file.go
                > vim ex 'e! file.go'
                > errlogmatch -wait 30s 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/file.go
                > vim ex 'w'
                > cmp file.go file.go.bad
                > vim expr 'getqflist()'
                [stdout]
                [{"bufnr":1,"col":6,"lnum":5,"module":"","nr":0,"pattern":"","text":"undeclared name: os","type":"","valid":1,"vcol":0}]
                > stdout '^\Q[{"bufnr":1,"col":1,"lnum":3,"module":"","nr":0,"pattern":"","text":"expected declaration, found blah","type":"","valid":1,"vcol":0}]\E$'
                FAIL: testdata/gofmt.txt:28: no match for `^\Q[{"bufnr":1,"col":1,"lnum":3,"module":"","nr":0,"pattern":"","text":"expected declaration, found blah","type":"","valid":1,"vcol":0}]\E$` found in stdout
                
FAIL
FAIL	github.com/myitcv/govim/cmd/govim	67.014s
?   	github.com/myitcv/govim/cmd/govim/config	[no test files]
?   	github.com/myitcv/govim/cmd/govim/internal/types	[no test files]
?   	github.com/myitcv/govim/internal/plugin	[no test files]
?   	github.com/myitcv/govim/internal/queue	[no test files]
?   	github.com/myitcv/govim/testdriver	[no test files]
?   	github.com/myitcv/govim/testsetup	[no test files]
Error on linue 34 in ./_scripts/dockerRun.sh
travis_time:end:29ff64a4:start=1564217394003368828,finish=1564217488821227530,duration=94817858702
The command "./_scripts/runDockerRun.sh" exited with 1.



Done. Your build exited with 1.
</pre></details>

Full travis log - successful run:
<details><pre>
heroku version
heroku/7.22.7 linux-x64 node-v11.10.1
imagemagick version
Version: ImageMagick 6.8.9-9 Q16 x86_64 2018-09-28 http://www.imagemagick.org
md5deep version
4.4
mercurial version
version 4.8
mysql version
mysql  Ver 14.14 Distrib 5.7.25, for Linux (x86_64) using  EditLine wrapper
openssl version
OpenSSL 1.0.2g  1 Mar 2016
packer version
1.3.3
postgresql client version
psql (PostgreSQL) 10.7 (Ubuntu 10.7-1.pgdg16.04+1)
ragel version
Ragel State Machine Compiler version 6.8 Feb 2013
sudo version
1.8.16
gzip version
gzip 1.6
zip version
Zip 3.0
vim version
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Nov 24 2016 16:44:48)
iptables version
iptables v1.6.0
curl version
curl 7.47.0 (x86_64-pc-linux-gnu) libcurl/7.47.0 GnuTLS/3.4.10 zlib/1.2.8 libidn/1.32 librtmp/2.3
wget version
GNU Wget 1.17.1 built on linux-gnu.
rsync version
rsync  version 3.1.1  protocol version 31
gimme version
v1.5.3
nvm version
0.34.0
perlbrew version
/home/travis/perl5/perlbrew/bin/perlbrew  - App::perlbrew/0.86
phpenv version
rbenv 1.1.2
rvm version
rvm 1.29.7 (latest) by Michal Papis, Piotr Kuczynski, Wayne E. Seguin [https://rvm.io]
default ruby version
ruby 2.5.3p105 (2018-10-18 revision 65156) [x86_64-linux]
CouchDB version
couchdb 1.6.1
ElasticSearch version
5.5.0
ravis_fold:start:worker_info
Worker information
hostname: 212d19e7-d169-471f-97a4-936203186951@1.worker-org-78c6f46d9c-htsrw.gce-production-1
version: v6.2.1 https://github.com/travis-ci/worker/tree/4e3246c044eb4915c2378ffacd0b3d3ed0136bba
instance: travis-job-cbe08ef4-84d8-4dba-a44b-8223e10eb13d travis-ci-sardonyx-xenial-1553530528-f909ac5 (via amqp)
startup: 6.725371367s
travis_fold:end:worker_info
travis_fold:start:system_info
Build system information
Build language: generic
Build group: stable
Build dist: xenial
Build id: 564325663
Job id: 564325674
Runtime kernel version: 4.15.0-1028-gcp
travis-build version: 24c88d12b
Build image provisioning date and time
Mon Mar 25 16:43:24 UTC 2019
Operating System Details
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.6 LTS
Release:	16.04
Codename:	xenial
Systemd Version
systemd 229
Cookbooks Version
42e42e4 https://github.com/travis-ci/travis-cookbooks/tree/42e42e4
git version
git version 2.21.0
bash version
GNU bash, version 4.3.48(1)-release (x86_64-pc-linux-gnu)
gcc version
gcc (Ubuntu 5.4.0-6ubuntu1~16.04.11) 5.4.0 20160609
docker version
Client:
 Version:           18.06.0-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        0ffa825
 Built:             Wed Jul 18 19:11:02 2018
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.0-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       0ffa825
  Built:            Wed Jul 18 19:09:05 2018
  OS/Arch:          linux/amd64
  Experimental:     false
clang version
clang version 7.0.0 (tags/RELEASE_700/final)
jq version
jq-1.5
bats version
Bats 0.4.0
shellcheck version
0.6.0
shfmt version
v2.6.3
ccache version
3.2.4
cmake version
cmake version 3.12.4
mInstalled Firefox version
firefox 63.0.1
MongoDB version
MongoDB 4.0.7
PhantomJS version
2.1.1
Pre-installed PostgreSQL versions
9.4.21
9.5.16
9.6.12
Redis version
redis-server 5.0.4
Pre-installed Go versions
1.11.1
mvn version
Apache Maven 3.6.0 (97c98ec64a1fdfee7767ce5ffb20918da4f719f3; 2018-10-24T18:41:47Z)
gradle version
Gradle 4.10.2!
lein version
Leiningen 2.9.1 on Java 11.0.2 OpenJDK 64-Bit Server VM
Pre-installed Node.js versions
v10.15.3
v11.0.0
v4.9.1
v6.17.0
v8.12.0
v8.15.1
v8.9
phpenv versions
  system
  5.6
  5.6.40
  7.1
  7.1.27
  7.2
* 7.2.15 (set by /home/travis/.phpenv/version)
  hhvm
  hhvm-stable
composer --version
Composer version 1.8.4 2019-02-11 10:52:10
Pre-installed Ruby versions
ruby-2.3.8
ruby-2.4.5
ruby-2.5.3
travis_fold:end:system_info

travis_fold:start:docker_mtu
travis_fold:end:docker_mtu
travis_fold:start:resolvconf
travis_fold:end:resolvconf

travis_fold:start:git.checkout
travis_time:start:026edd60
$ git clone --depth=50 --branch=master https://github.com/myitcv/govim.git myitcv/govim
Cloning into 'myitcv/govim'...
travis_time:end:026edd60:start=1564217685184703884,finish=1564217686049844718,duration=865140834
$ cd myitcv/govim
$ git checkout -qf 7034f5a67ee5043c50231e15d78d4cf23c8ca614
travis_fold:end:git.checkout


Setting environment variables from repository settings
$ export GH_TOKEN=[secure]
$ export GH_USER=myitcv

Setting environment variables from .travis.yml
$ export GO111MODULE=on
$ export GOPROXY=https://proxy.golang.org
$ export GO_VERSION="go1.12.7"
$ export VIM_FLAVOR="gvim"
$ export VIM_VERSION="v8.1.1657"
$ export VIM_COMMAND="xvfb-run -a gvim -f"

$ bash -c 'echo $BASH_VERSION'
4.3.48(1)-release

travis_fold:start:before_install.1
travis_time:start:0a487b5c
$ docker --version
Docker version 18.06.0-ce, build 0ffa825
travis_time:end:0a487b5c:start=1564217686092184742,finish=1564217686390888754,duration=298704012
travis_fold:end:before_install.1
travis_fold:start:before_install.2
travis_time:start:038d7e87
$ ./_scripts/buildGovimImage.sh
Sending build context to Docker daemon  2.048kB


Step 1/11 : FROM govim/govim:go1.12.7_gvim_v8.1.1657_v1
go1.12.7_gvim_v8.1.1657_v1: Pulling from govim/govim
898c46f3b1a1: Pulling fs layer
63366dfa0a50: Pulling fs layer
041d4cd74a92: Pulling fs layer
6e1bee0f8701: Pulling fs layer
930c33e06246: Pulling fs layer
c2eb65575a45: Pulling fs layer
0adb2cbf93a0: Pulling fs layer
0650c35c0db6: Pulling fs layer
7c859565a35f: Pulling fs layer
ce8f93ab5d45: Pulling fs layer
27e2dca98e6c: Pulling fs layer
36b19e4bf3f2: Pulling fs layer
afd947b29ffc: Pulling fs layer
df7a3dc01368: Pulling fs layer
2c21a8530cff: Pulling fs layer
ea566e43f97e: Pulling fs layer
6e1bee0f8701: Waiting
930c33e06246: Waiting
c2eb65575a45: Waiting
0adb2cbf93a0: Waiting
0650c35c0db6: Waiting
7c859565a35f: Waiting
ce8f93ab5d45: Waiting
27e2dca98e6c: Waiting
36b19e4bf3f2: Waiting
afd947b29ffc: Waiting
df7a3dc01368: Waiting
2c21a8530cff: Waiting
ea566e43f97e: Waiting
63366dfa0a50: Verifying Checksum
63366dfa0a50: Download complete
041d4cd74a92: Verifying Checksum
041d4cd74a92: Download complete
6e1bee0f8701: Verifying Checksum
6e1bee0f8701: Download complete
898c46f3b1a1: Verifying Checksum
898c46f3b1a1: Download complete
930c33e06246: Verifying Checksum
930c33e06246: Download complete
c2eb65575a45: Verifying Checksum
c2eb65575a45: Download complete
0adb2cbf93a0: Verifying Checksum
0adb2cbf93a0: Download complete
ce8f93ab5d45: Verifying Checksum
ce8f93ab5d45: Download complete
0650c35c0db6: Verifying Checksum
0650c35c0db6: Download complete
7c859565a35f: Verifying Checksum
7c859565a35f: Download complete
afd947b29ffc: Verifying Checksum
afd947b29ffc: Download complete
36b19e4bf3f2: Verifying Checksum
36b19e4bf3f2: Download complete
898c46f3b1a1: Pull complete
df7a3dc01368: Verifying Checksum
df7a3dc01368: Download complete
63366dfa0a50: Pull complete
27e2dca98e6c: Verifying Checksum
27e2dca98e6c: Download complete
041d4cd74a92: Pull complete
6e1bee0f8701: Pull complete
ea566e43f97e: Verifying Checksum
ea566e43f97e: Download complete
930c33e06246: Pull complete
c2eb65575a45: Pull complete
2c21a8530cff: Verifying Checksum
2c21a8530cff: Download complete
0adb2cbf93a0: Pull complete
0650c35c0db6: Pull complete
7c859565a35f: Pull complete
ce8f93ab5d45: Pull complete
27e2dca98e6c: Pull complete
36b19e4bf3f2: Pull complete
afd947b29ffc: Pull complete
df7a3dc01368: Pull complete
2c21a8530cff: Pull complete
ea566e43f97e: Pull complete
Digest: sha256:d8a9e417d9b273970a5d76e687ddceec2961236fbc49318bc0e945b8e9997a16
Status: Downloaded newer image for govim/govim:go1.12.7_gvim_v8.1.1657_v1
 ---> 35bfc91d63e5
Step 2/11 : ARG USER
 ---> Running in ff7bea04c650
Removing intermediate container ff7bea04c650
 ---> 5847b61859e0
Step 3/11 : ARG UID
 ---> Running in ce4aa92ca45e
Removing intermediate container ce4aa92ca45e
 ---> 4fec84ef99b0
Step 4/11 : ARG GID
 ---> Running in d76dbf5b0fad
Removing intermediate container d76dbf5b0fad
 ---> 53d42504cd72
Step 5/11 : ENV PATH=/vbash/bin:/home/$USER/.local/bin:$PATH
 ---> Running in b6aac83a5244
Removing intermediate container b6aac83a5244
 ---> d05d7ae8f9fd
Step 6/11 : ENV GOPATH=/home/$USER/gopath
 ---> Running in 9e168f7c29bd
Removing intermediate container 9e168f7c29bd
 ---> f0fe0401d2da
Step 7/11 : RUN sh -c "if ! getent group $GID; then groupadd -g $GID $USER; fi" &&     adduser --uid $UID --gid $GID --disabled-password --gecos "" $USER
 ---> Running in 00cec71c174b
Adding user `travis' ...
Adding new user `travis' (2000) with group `travis' ...
Creating home directory `/home/travis' ...
Copying files from `/etc/skel' ...
Removing intermediate container 00cec71c174b
 ---> 59c9b89369f2
Step 8/11 : RUN usermod -aG sudo $USER
 ---> Running in 40e9f105251f
Removing intermediate container 40e9f105251f
 ---> 74073bfcec95
Step 9/11 : RUN echo "$USER ALL=(ALL:ALL) NOPASSWD: ALL" > /etc/sudoers.d/$USER
 ---> Running in 2d2eb1b6c270
Removing intermediate container 2d2eb1b6c270
 ---> 4c924d3754bd
Step 10/11 : WORKDIR /home/$USER
 ---> Running in 2678315badea
Removing intermediate container 2678315badea
 ---> 240d7dff8378
Step 11/11 : USER $USER
 ---> Running in a5105c1c073c
Removing intermediate container a5105c1c073c
 ---> 303bdfe31f6b
Successfully built 303bdfe31f6b
Successfully tagged govim:latest
travis_time:end:038d7e87:start=1564217686399409292,finish=1564217749126515933,duration=62727106641
travis_fold:end:before_install.2
travis_time:start:26a4cfe6
$ ./_scripts/runDockerRun.sh
> source "${BASH_SOURCE%/*}/common.bash"
> eval "VIM_COMMAND=\"\$DEFAULT_${VIM_FLAVOR^^}_COMMAND\""
> cat <<EOD
Environment is:
  VIM_FLAVOR=$VIM_FLAVOR
  VIM_COMMAND=$VIM_COMMAND
EOD
Environment is:
  VIM_FLAVOR=gvim
  VIM_COMMAND=xvfb-run -a gvim -f
> echo -e "machine api.github.com\n  login $GH_USER\n  password $GH_TOKEN" >>~/.netrc
> echo -e "machine github.com\n  login $GH_USER\n  password $GH_TOKEN" >>~/.netrc
> echo -e "machine githubusercontent.com\n  login $GH_USER\n  password $GH_TOKEN" >>~/.netrc
> go version
go version go1.12.7 linux/amd64
> vim --version
VIM - Vi IMproved 8.1 (2018 May 18, compiled Jul 18 2019 08:20:25)
Included patches: 1-1657
Compiled by root@3ccfdb568c5f
Huge version with GTK2 GUI.  Features included (+) or not (-):
+acl               -farsi             -mouse_sysmouse    -tag_any_white
+arabic            +file_in_path      +mouse_urxvt       -tcl
+autocmd           +find_in_path      +mouse_xterm       +termguicolors
+autochdir         +float             +multi_byte        +terminal
-autoservername    +folding           +multi_lang        +terminfo
+balloon_eval      -footer            -mzscheme          +termresponse
+balloon_eval_term +fork()            -netbeans_intg     +textobjects
+browse            +gettext           +num64             +textprop
++builtin_terms    -hangul_input      +packages          +timers
+byte_offset       +iconv             +path_extra        +title
+channel           +insert_expand     -perl              +toolbar
+cindent           +job               +persistent_undo   +user_commands
+clientserver      +jumplist          +postscript        +vartabs
+clipboard         +keymap            +printer           +vertsplit
+cmdline_compl     +lambda            +profile           +virtualedit
+cmdline_hist      +langmap           -python            +visual
+cmdline_info      +libcall           -python3           +visualextra
+comments          +linebreak         +quickfix          +viminfo
+conceal           +lispindent        +reltime           +vreplace
+cryptv            +listcmds          +rightleft         +wildignore
+cscope            +localmap          -ruby              +wildmenu
+cursorbind        -lua               +scrollbind        +windows
+cursorshape       +menu              +signs             +writebackup
+dialog_con_gui    +mksession         +smartindent       +X11
+diff              +modify_fname      -sound             -xfontset
+digraphs          +mouse             +spell             +xim
+dnd               +mouseshape        +startuptime       -xpm
-ebcdic            +mouse_dec         +statusline        +xsmp_interact
+emacs_tags        -mouse_gpm         -sun_workshop      +xterm_clipboard
+eval              -mouse_jsbterm     +syntax            -xterm_save
+ex_extra          +mouse_netterm     +tag_binary        
+extra_search      +mouse_sgr         -tag_old_static    
   system vimrc file: "$VIM/vimrc"
     user vimrc file: "$HOME/.vimrc"
 2nd user vimrc file: "~/.vim/vimrc"
      user exrc file: "$HOME/.exrc"
  system gvimrc file: "$VIM/gvimrc"
    user gvimrc file: "$HOME/.gvimrc"
2nd user gvimrc file: "~/.vim/gvimrc"
       defaults file: "$VIMRUNTIME/defaults.vim"
    system menu file: "$VIMRUNTIME/menu.vim"
  fall-back for $VIM: "/vim/share/vim"
Compilation: gcc -c -I. -Iproto -DHAVE_CONFIG_H -DFEAT_GUI_GTK  -pthread -I/usr/include/gtk-2.0 -I/usr/lib/x86_64-linux-gnu/gtk-2.0/include -I/usr/include/gio-unix-2.0/ -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libpng16 -I/usr/include/pango-1.0 -I/usr/include/harfbuzz -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/freetype2 -I/usr/include/libpng16   -g -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1       
Linking: gcc   -L/usr/local/lib -Wl,--as-needed -o vim   -lgtk-x11-2.0 -lgdk-x11-2.0 -lpangocairo-1.0 -latk-1.0 -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lpangoft2-1.0 -lpango-1.0 -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lSM -lICE -lXt -lX11 -lXdmcp -lSM -lICE  -lm -ltinfo -lnsl  -ldl           
> ./_scripts/revendorToolsInternal.sh
	> source "${BASH_SOURCE%/*}/common.bash"
	> shopt -s globstar
	> shopt -s extglob
	> go mod download
	go: finding golang.org/x/tools v0.0.0-20190716221150-e98af2309876
	go: finding gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
	go: finding github.com/kr/pty v1.1.5
	go: finding github.com/fsnotify/fsnotify v1.4.7
	go: finding golang.org/x/sync v0.0.0-20190423024810-112230192c58
	go: finding golang.org/x/tools/gopls v0.1.4-0.20190716221150-e98af2309876
	go: finding github.com/rogpeppe/go-internal v1.3.0
	go: finding github.com/acarl005/stripansi v0.0.0-20180116102854-5a71ef0e047d
	go: finding github.com/kr/pretty v0.1.0
	go: finding golang.org/x/sys v0.0.0-20190429190828-d89cdac9e872
	go: finding gopkg.in/retry.v1 v1.0.3
	go: finding github.com/myitcv/vbash v0.0.4
	go: finding golang.org/x/tools v0.0.0-20190710153321-831012c29e42
	go: finding gopkg.in/errgo.v2 v2.1.0
	go: finding honnef.co/go/tools v0.0.0-20190614002413-cb51c254f01b
	go: finding golang.org/x/net v0.0.0-20190620200207-3b0461eec859
	go: finding github.com/kr/text v0.1.0
	go: finding github.com/rogpeppe/clock v0.0.0-20190514195947-2896927a307a
	go: finding github.com/frankban/quicktest v1.2.2
	go: finding gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127
	go: finding golang.org/x/mod v0.0.0-20190513183733-4bf6d317e70e
	go: finding github.com/google/renameio v0.1.0
	go: finding golang.org/x/text v0.3.0
	go: finding github.com/kisielk/gotool v1.0.0
	go: finding github.com/rogpeppe/go-internal v1.2.1
	go: finding github.com/BurntSushi/toml v0.3.1
	go: finding mvdan.cc/sh/v3 v3.0.0-alpha1
	go: finding golang.org/x/sys v0.0.0-20190215142949-d0b11bdaac8a
	go: finding github.com/kr/pty v1.1.1
	go: finding golang.org/x/crypto v0.0.0-20190510104115-cbcb75029529
	go: finding golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2
	go: finding golang.org/x/tools v0.0.0-20190530171427-2b03ca6e44eb
	go: finding github.com/google/go-cmp v0.2.1-0.20190312032427-6f77996f0c42
	go: finding golang.org/x/crypto v0.0.0-20181106171534-e4dc69e5b2fd
	go: finding golang.org/x/sys v0.0.0-20181107165924-66b7b1311ac8
	go: finding golang.org/x/sync v0.0.0-20181108010431-42b317875d0f
	go: finding golang.org/x/net v0.0.0-20190311183353-d8887717615a
	go: finding golang.org/x/net v0.0.0-20190404232315-eb5bcb51f2a3
	go: finding golang.org/x/sys v0.0.0-20190412213103-97732733099d
	> tools=$(go list -m -f={{.Dir}} golang.org/x/tools)
	> echo "Tools is $tools"
	Tools is /home/travis/gopath/pkg/mod/golang.org/x/tools@v0.0.0-20190716221150-e98af2309876
	> cd $(git rev-parse --show-toplevel)
	> rsync -a --delete --chmod=D0755,F0644 $tools/internal/ ./cmd/govim/internal/golang_org_x_tools
	> sed -i 's+golang.org/x/tools/internal+github.com/myitcv/govim/cmd/govim/internal/golang_org_x_tools+g' ./cmd/govim/internal/golang_org_x_tools/**/*.go
	> rm ./cmd/govim/internal/golang_org_x_tools/**/*_test.go
	> rm -f ./cmd/govim/internal/golang_org_x_tools/LICENSE
	> cp $tools/LICENSE ./cmd/govim/internal/golang_org_x_tools
	> go mod tidy
> go install golang.org/x/tools/gopls
> rm -f $(git ls-files -- ':!:cmd/govim/internal/golang_org_x_tools' '**/gen_*.*' 'gen_*.*') .travis.yml
> go generate $(go list ./... | grep -v 'govim/internal/golang_org_x_tools')
> go test $(go list ./... | grep -v 'govim/internal/golang_org_x_tools')
ok  	github.com/myitcv/govim	10.030s
ok  	github.com/myitcv/govim/cmd/govim	62.756s
?   	github.com/myitcv/govim/cmd/govim/config	[no test files]
?   	github.com/myitcv/govim/cmd/govim/internal/types	[no test files]
?   	github.com/myitcv/govim/internal/plugin	[no test files]
?   	github.com/myitcv/govim/internal/queue	[no test files]
?   	github.com/myitcv/govim/testdriver	[no test files]
?   	github.com/myitcv/govim/testsetup	[no test files]
> go test -race $(go list ./... | grep -v 'govim/internal/golang_org_x_tools')
ok  	github.com/myitcv/govim	24.217s
ok  	github.com/myitcv/govim/cmd/govim	151.512s
?   	github.com/myitcv/govim/cmd/govim/config	[no test files]
?   	github.com/myitcv/govim/cmd/govim/internal/types	[no test files]
?   	github.com/myitcv/govim/internal/plugin	[no test files]
?   	github.com/myitcv/govim/internal/queue	[no test files]
?   	github.com/myitcv/govim/testdriver	[no test files]
?   	github.com/myitcv/govim/testsetup	[no test files]
> go vet $(go list ./... | grep -v 'govim/internal/golang_org_x_tools')
> go run honnef.co/go/tools/cmd/staticcheck $(go list ./... | grep -v 'govim/internal/golang_org_x_tools')
> go mod tidy
> go list all >/dev/null
> diff <(echo -n) <(gofmt -d $(git ls-files '**/*.go' '*.go' | grep -v cmd/govim/internal/golang_org_x_tools))
> test -z "$(git status --porcelain)" || (
	git status
	git diff
	false
)
travis_time:end:26a4cfe6:start=1564217749134258837,finish=1564218019900049764,duration=270765790927
The command "./_scripts/runDockerRun.sh" exited with 0.



Done. Your build exited with 0.
</pre></details>

Also happened during test of https://github.com/myitcv/govim/commit/54b65c8c61dc92ac9eea331d9d9a077784a56463.

Job:
<pre>GO_VERSION="go1.13beta1" VIM_FLAVOR="gvim" VIM_VERSION="v8.1.1657" VIM_COMMAND="xvfb-run -a gvim -f"</pre>

<pre>
> go test $(go list ./... | grep -v 'govim/internal/golang_org_x_tools')
ok  	github.com/myitcv/govim	10.640s
--- FAIL: TestScripts (23.02s)
    --- FAIL: TestScripts/scripts (0.00s)
        --- FAIL: TestScripts/scripts/gofmt (1.24s)
            testscript.go:308: 
                # Test that govim#config#Set("FormatOnSave", "gofmt") and GOVIMGoFmt work
                # :GOVIMGoFmt whole file (0.301s)
                # Format on save (0.112s)
                # Format on save (bad syntax) (0.110s)
                > cp file.go.bad file.go
                > vim ex 'e! file.go'
                > errlogmatch -wait 30s 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/file.go
                > vim ex 'w'
                > cmp file.go file.go.bad
                > vim expr 'getqflist()'
                [stdout]
                [{"bufnr":1,"col":6,"lnum":5,"module":"","nr":0,"pattern":"","text":"undeclared name: os","type":"","valid":1,"vcol":0}]
                > stdout '^\Q[{"bufnr":1,"col":1,"lnum":3,"module":"","nr":0,"pattern":"","text":"expected declaration, found blah","type":"","valid":1,"vcol":0}]\E$'
                FAIL: testdata/gofmt.txt:28: no match for `^\Q[{"bufnr":1,"col":1,"lnum":3,"module":"","nr":0,"pattern":"","text":"expected declaration, found blah","type":"","valid":1,"vcol":0}]\E$` found in stdout
                
FAIL
</pre>

--- a/cmd/govim/testdata/gofmt.txt
+++ b/cmd/govim/testdata/gofmt.txt
@@ -1,10 +1,15 @@
 # Test that govim#config#Set("FormatOnSave", "gofmt") and GOVIMGoFmt work
 
+# Notice below: file.go and file.go.orig are identical. This is so that we
+# don't need to copy file.go.orig to file.go before opening file.go Because
+# doing so triggers govim to consider the file change from a file watcher
+# perspective which throws the observed diagnostics etc.
+
 # :GOVIMGoFmt whole file
-cp file.go.orig file.go
 vim ex 'e! file.go'
 errlogmatch -wait 30s 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/file.go
 vim ex 'GOVIMGoFmt'
+errlogmatch -wait 30s 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/file.go
 vim ex 'noautocmd w'
 cmp file.go file.go.gofmt
 
@@ -14,6 +19,7 @@ vim call 'govim#config#Set' '["FormatOnSave", "gofmt"]'
 vim ex 'e! file.go'
 errlogmatch -wait 30s 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/file.go
 vim ex 'w'
+errlogmatch -wait 30s 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/file.go
 cmp file.go file.go.gofmt
 
 errlogmatch -count=0 'LogMessage callback: &protocol\.LogMessageParams\{Type:%v, Message:".*'
@@ -21,7 +27,7 @@ errlogmatch -count=0 'LogMessage callback: &protocol\.LogMessageParams\{Type:%v,
 # Format on save (bad syntax)
 cp file.go.bad file.go
 vim ex 'e! file.go'
-errlogmatch -wait 30s 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/file.go
+errlogmatch -wait 30s 'sendJSONMsg: \[0,\[\d+,"call","setqflist".*'
 vim ex 'w'
 cmp file.go file.go.bad
 vim expr 'getqflist()'
@@ -45,11 +51,18 @@ errlogmatch -count=0 'LogMessage callback: &protocol\.LogMessageParams\{Type:%v,
 -- go.mod --
 module mod.com
 
+-- file.go --
+package blah
+
+const ( x = 5
+y = x
+ )
+
 -- file.go.orig --
 package blah
 
 const ( x = 5
-y = os.PathSeparator
+y = x
  )
 
 -- file.go.bad --
@@ -61,5 +74,5 @@ package blah
 
 const (
 	x = 5
-	y = os.PathSeparator
+	y = x
 )

--- a/cmd/govim/testdata/goimports.txt
+++ b/cmd/govim/testdata/goimports.txt
@@ -1,14 +1,19 @@
 # Test that govim#config#Set("FormatOnSave", "goimports") and GOVIMGoImports work
 
+# Notice below: file.go and file.go.orig are identical. This is so that we
+# don't need to copy file.go.orig to file.go before opening file.go Because
+# doing so triggers govim to consider the file change from a file watcher
+# perspective which throws the observed diagnostics etc.
+
 # Ensure the default is goimports
 vim expr 'govim#config#Get().FormatOnSave'
 stdout '^"goimports"$'
 
 # :GOVIMGoImports whole file
-cp file.go.orig file.go
 vim ex 'e! file.go'
 errlogmatch -wait 30s 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/file.go
 vim ex 'GOVIMGoImports'
+errlogmatch -wait 30s 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/file.go
 vim ex 'noautocmd w'
 cmp file.go file.go.goimports
 
@@ -18,6 +23,7 @@ vim call 'govim#config#Set' '["FormatOnSave", "goimports"]'
 vim ex 'e! file.go'
 errlogmatch -wait 30s 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/file.go
 vim ex 'w'
+errlogmatch -wait 30s 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/file.go
 cmp file.go file.go.goimports
 
 errlogmatch -count=0 'LogMessage callback: &protocol\.LogMessageParams\{Type:%v, Message:".*'
@@ -25,7 +31,7 @@ errlogmatch -count=0 'LogMessage callback: &protocol\.LogMessageParams\{Type:%v,
 # Format on save (bad syntax)
 cp file.go.bad file.go
 vim ex 'e! file.go'
-errlogmatch -wait 30s 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/file.go
+errlogmatch -wait 30s 'sendJSONMsg: \[0,\[\d+,"call","setqflist".*'
 vim ex 'w'
 cmp file.go file.go.bad
 vim expr 'getqflist()'
@@ -47,6 +53,13 @@ errlogmatch -count=0 'LogMessage callback: &protocol\.LogMessageParams\{Type:%v,
 
 -- go.mod --
 module mod.com
+
+-- file.go --
+package blah
+
+const ( x = 5
+y = os.PathSeparator
+ )
 
 -- file.go.orig --
 package blah

